VerilogCodingStyle.md

```diff
@@ -2291,6 +2292,109 @@ always_comb begin
 end
 ```
 
+### Functions and Tasks
+
+The following section applies to synthesizable RTL only.
```

Comment:
Once the DV style guide stuff is merged this should refer to the guidance for functions on DV. I'll await that PR to go in before this is merged and add a reference.

Reply:
this has landed now if you want to add a reference.